src/components/App.js

```diff
@@ -1,8 +1,9 @@
 import React from 'react';
+import Routes from './Routes';
 
 const App = () => (
   <div>
```

Comment:
You can use `React.Fragment` here so as not to render the extra div in the dom.

Reply:
@adesege worked on feedback